mysql-test/main/group_min_max_innodb.result

```diff
@@ -323,7 +323,7 @@ JOIN t1 ON dt.a=t1.b;
 id	select_type	table	type	possible_keys	key	key_len	ref	rows	Extra
 1	PRIMARY	t1	ALL	NULL	NULL	NULL	NULL	#	Using where
 1	PRIMARY	<derived2>	ref	key0	key0	6	test.t1.b	#	
-2	DERIVED	t2	range	a	a	58	NULL	#	Using index for group-by; Using temporary; Using filesort
+2	DERIVED	t2	index	NULL	a	6	NULL	#	Using index; Using temporary; Using filesort
```

Comment:
Why this change?  The fix was expected to only disable Split-Materialized. This query doesn't use it...